src/getHumanAge.test.js

```diff
@@ -7,4 +7,52 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return a correct dog and cats years for an old animals', () => {
```

Comment:
The test description 'should return a correct dog and cats years for an old animals' is unclear. Consider rephrasing it to something like 'should return correct human age equivalents for old animals'.